time.c

```diff
@@ -57,6 +57,25 @@ time_str(u_long x, bool iso8601fmt) {
 	return ret;
 }
 
+/* timeval_str -- format one timestamp (NULL means current time) and return it
```

Comment:
add example of the output, in particular, showing the three digit milliseconds time precision.